src/scss/constants.scss

```diff
@@ -1,11 +1,13 @@
 $primaryColor: var(--primary-color);
```

Comment:
the core changes are in `src/scss/constants.scss` 
all other files contain color renaming changes